cannon/README.md

```diff
@@ -19,18 +19,18 @@ For more information, see [Docs](./docs/README.md).
 
 ## Usage
 
-```shell
-# Build op-program server-mode and MIPS-client binaries.
-cd ../op-program
-make op-program # build
+The following usage requires `docker` with buildkit enabled to generate
+a reproducible absolute prestate with the [op-program](../op-program/).
 
-# Switch back to cannon, and build the CLI
-cd ../cannon
-make cannon
+The reproducible absolute prestate is built in a docker container with:
+- golang version `1.21.3`
+- alpine `3.18`
 
+```shell
+# Build op-program server-mode and MIPS-client binaries.
 # Transform MIPS op-program client binary into first VM state.
 # This outputs state.json (VM state) and meta.json (for debug symbols).
-./bin/cannon load-elf --path=../op-program/bin/op-program-client.elf
+cd .. && make reproducible-prestate
```

Comment:
I think I'd scrap this change and leave the cannon description as more generic.  This is really about how to use cannon and the op-program is used as an example.  People will want to be able to read this to work out how to load their own programs into cannon, not just op-program.